### ISSUE TITLE
Moniko(t) käännöksissä

### ISSUE THREAD

**Issue by ossimantylahti** [OCA-Finland/odoo-fi#12]
Yleinen kielenhuoltolinjaus. Odoossa on aika monessa paikassa yksikkö tai monikko siten, että monikko on suluissa. Esimerkiksi
customer(s), invoice(s)
Olen kääntänhyt nämä kaikki järjestään suomeksi vain monikkomuotoon. Siis asiakkaat, laskut, jne. Tämä on pienempi paha kuin kääntää joka ikinen "lasku tai laskut". Tästä nimittäin seuraa hiton pitkiä lauseita. Kun englanniksi sanotaan "Following invoice(s) have problems:" niin tiukasti ottaen tämä kääntyisi suomeksi "Seuraavassa laskussa tai laskuissa on virheitä:"
Jos käyttäisin pelkkää yksikköä, virhe pistää silmään. 
"Seuraavassa laskussa on virhe:
INV/2024/0023, INV/2024/0024, INV/2024/0038"
Sen sijaan monikkoa käytettäessä tämä on ok:
"Seuraavissa laskuissa on virhe: 
INV/2024/00023"
Neljäs vaihtoehto olisi kääntää "Seuraavassa (seuraavissa) laskussa (laskuissa) on virhe..." tai "Seuraavassa (-issa) laskussa (-issa) on virhe". Mutta tämä on todella kömpelön näköistä.

**Comment by ossimantylahti** [comment]
OCA 1.3.25 hyväksytty

**Comment by jusle** [comment]
Kyllä, tämä pitäisi ratkaista ohjelmallisesti, jotta menisi tasan tarkkaan oikein, joten monikko paras ratkaisu.
